app/components/prod.min.js

```diff
@@ -0,0 +1,148 @@
+        const f =
+          "<style>.bmc-btn svg {\n    height: 32px !important;\n    margin-bottom: 0px !important;\n    box-shadow: none !important;\n    border: none !important;\n    vertical-align: middle !important;\n    transform: scale(0.9);\n    flex-shrink: 0;\n}\n\n.bmc-btn {\n    min-width: 210px;\n    color: [FONT_COLOR];\n    background-color: [BG_COLOR] !important;\n    height: 60px;\n    border-radius: 12px;\n    font-size: [FONT_SIZE];\n    font-weight: [FONT_WEIGHT];\n    border: none;\n    padding: 0px 24px;\n    line-height: 27px;\n    text-decoration: none !important;\n    display: inline-flex !important;\n    align-items: center;\n    font-family: [FONT] !important;\n    -webkit-box-sizing: border-box !important;\n    box-sizing: border-box !important;\n}\n\n.bmc-btn:hover, .bmc-btn:active, .bmc-btn:focus {\n    text-decoration: none !important;\n    cursor: pointer;\n}\n\n.bmc-btn-text {\n   margin-left: 8px;\n   display: inline-block;\n   line-height: 0;\n   width: 100%;\n   flex-shrink: 0;\n   font-family: [FONT] !important;\n}\n\n.logo-outline {\n    fill: [OUTLINE_COLOR];\n}\n\n.logo-coffee {\n    fill: [COFFEE_COLOR];\n}</style>";
+        let c = `<link href="https://fonts.googleapis.com/css?family=${o[C].google_font_name}&display=swap" rel="stylesheet">`,
+          s = "28px",
+          p = "Normal";
+        ("Arial" !== o[C].google_font_name &&
+          "Lato" !== o[C].google_font_name &&
+          "Comic" !== o[C].google_font_name &&
+          "Inter" !== o[C].google_font_name &&
+          "Bree+Serif" !== o[C].google_font_name &&
+          "Poppins" !== o[C].google_font_name) ||
+          ((s = "24px"), (p = "Bold"));
+        const m =
+          c +
+          '<div class="bmc-btn-container"><a class="bmc-btn" ' +
+          `target="_blank" href="https://buymeacoffee.com/${n}">` +
+          `${a}<span class="bmc-btn-text">${t}</span>` +
+          "</a></div>";
+        return (
+          f
+            .replace("[BG_COLOR]", e)
+            .replace("[FONT]", o[C].css_font_name)
+            .replace("[FONT_COLOR]", r)
+            .replace("[FONT_SIZE]", s)
+            .replace("[FONT_WEIGHT]", p)
+            .replace("[OUTLINE_COLOR]", i)
+            .replace("[COFFEE_COLOR]", l) + m
+        );
+      };
+    let a = document.querySelector('script[data-name="bmc-button"]');
+    if (a) {
+      a.nextElementSibling.innerHTML = bmcBtnWidget(
```

Comment:
Original code uses `document.writeln` which is not longer recommend and also have a lot of security issues.
I just download it, update it to use good old `innerHTML` in a empty div :)